README.md

```diff
@@ -241,6 +241,30 @@ function handleSelection(value, event) {
 }
 ```
 
+**onChange** { Function }: A callback to run when the menu is opened or closed. It will be passed the `isOpen` state of the menu, e.g. `{ isOpen: true }`
```

Comment:
I don't think `onChange` is a clear and specific name for this prop. `onMenuToggle` might be more clear.

Is there a reason to have one prop function that receives an object with state instead of two such functions: `onMenuOpen` and `onMenuClose`?